kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/Food2Door/suppliers/HealthyShop.java

```diff
@@ -0,0 +1,18 @@
+package com.kodilla.good.patterns.challenges.Food2Door.suppliers;
+
+import com.kodilla.good.patterns.challenges.Food2Door.delivery.DeliveryRequest;
+
+public class HealthyShop implements Supplier {
+    private String name = "HealthyShop";
+    private String regon = "12122112";
```

Comment:
name i regon to stale - powinny byc private final static

Reply:
poprawione